### PR TITLE
Deployment infrastructure

### LINKED ISSUE #40: Automated deployment instructions
Need to migrate the currently used autodeployment instructions to `Jenkinsfile`, and update them to deploy the new site.

### PR DESCRIPTION
**Pull request by ForNeVeR**
I've set up the deployment script, and also added a bit of deployment documentation.

All the scripts are fully tested.

Closes #40.

### CODE REVIEW

File: scripts/stop.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sudo /sbin/stop cor-site
+cd /opt/codingteam/codingteam.org.ru/
+rm -rf *
```
Comment (Minoru) [inline scripts/stop.sh]: I don't know how this script is executed, so I'm assuming it's simply `/bin/sh scripts/stop.sh`. Correct me if I'm wrong.

By default, shells carry on with the operations regardless of errors. If `cd` above fails, shell will print out an error and *proceed to running `rm -rf *` in the current directory*.

There are two ways around that:
1. `set -e` at the beginning of the script. (See [POSIX Shell spec](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#set) for details on that one.)
2. Combine `cd` and `rm` with `&&`: `cd /opt/codingteam/codingteam.org.ru/ && rm -rf *`.

And if you ask me, I'd vote for using *both* approaches here.

Comment (ForNeVeR) [inline scripts/stop.sh]: That's a very good suggestion, thank you!

File: docs/deployment.md [hunk 2]
```diff
@@ -30,10 +30,10 @@ maintainer decides it's time to release.
 Make sure it's installed and all dependencies are met. Consult
 `scripts/Jenkinsfile.deploy` and set up all the build parameters on your server.
 
-The deployment is performed over SSH, so make sure that CI have SSH access to
-the target machine as the same `cor-site` user and a permission to execute
-`sudo /sbin/stop cor-site` and `sudo /sbin/start cor-site` commands (and nothing
-else).
+The deployment is performed over SSH, so make sure that CI server have SSH
+access to the target machine as the same `cor-site` user, and that `cor-site`
+have a permission to execute the commands `sudo /sbin/stop cor-site` and
```
Comment (Minoru) [inline docs/deployment.md]: `cor-site` is a "user", so he "has" permission. I also don't think the article is needed here. So it all should read "…and `cor-site` has permission to execute…".

File: scripts/stop.sh (hunk 1 above)
Comment (ForNeVeR) [inline scripts/stop.sh]: Actually, it was easier: just run `rm -rf /opt/codingteam/codingteam.org.ru/*`.

**Comment by ForNeVeR** [comment]
Thank you for review, guys.

**Pull request merged**